CMakeLists.txt.in

```diff
@@ -294,7 +294,7 @@ ExternalProject_Add(seastar
     -DSeastar_TESTING=OFF
     -DSeastar_CXX_FLAGS=-Wno-error
     -DSeastar_LD_FLAGS=-pthread
-    -DSeastar_API_LEVEL=1
+    -DSeastar_API_LEVEL=2
```

Comment:
this is fantastic! i cannot believe smf still had the compat namespace. perfect. thank you.